app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html

```diff
@@ -0,0 +1,173 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.utils.Dates
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_nav
+
+@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None) {
+    @defining(c.application.asBTI) { btiApp =>
+
+        <h1 class="heading-large">#@c.reference - @btiApp.getType for @btiApp.holder.traderName</h1>
+
+        <div class="grid-row">
+            <div class="column-one-quarter">
+                @case_nav("ruling", c.reference)
+            </div>
+
+            <div class="column-three-quarters">
+
+                @if(c.decision.isDefined) {
+
+                    <div class="grid-row">
+                        <div id="ruling" class="column-one-third">
+                            <h4 class="heading-medium mt-0">Ruling</h4>
+                        </div>
+                        <div id="change-details" class="column-two-thirds">
+                            <p><a id="change" href="#">Change</a></p>
```

Comment:
If we aren't displaying incomplete functionality we should probably comment this for now